### PR TITLE
fix: Added readPreference requirements for Git Import

### PR DESCRIPTION
**Pull request by infinitetrooper**
This is required for Git Import to work in scenarios of external MongoDB configurations

https://theappsmith.slack.com/archives/C0341RERY4R/p1678810579003419

### CODE REVIEW

File: website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ The exact URI to be set here can be obtained from MongoDB Cloud's web interface.
 
 :::note Important
 * Use a MongoDB database with Appsmith, `replicaSet` needs to be configured on the database. This can be done by connecting to the database as `admin`, and running [`rs.initiate()`](https://docs.mongodb.com/manual/reference/method/rs.initiate/). If you are using MongoDB Cloud, this is always done for you.
```
Comment (nayan-rafiq) [inline website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md:23]: > Use a MongoDB database with Appsmith

I guess it should be: To use an existing MongoDB database with Appsmith

File: website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ The exact URI to be set here can be obtained from MongoDB Cloud's web interface.
...
-
 * Ensure that the credentials used to connect to MongoDB should have the `readWrite` and `clusterMonitor` roles assigned.
+* In a multi-node configuration of MongoDB cluster, the `readPreference=primary` URL parameter needs to be added to the MongoDB connection string. This is required to support app Import-Export feature.
```
Comment (nayan-rafiq) [inline website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md:25]: > This is required to support app Import-Export feature.
I think we can rephrase this to the following:
This is required to support transactional operations in MongoDB

Also can we add a sample connection URL as well here for better understanding. Here is a sample connection string: 
`mongodb://localhost:27017/appsmith?authSource=admin&replicaSet=rs0&readPreference=primary`

**Review by nayan-rafiq — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md (hunk 1 above)
Comment (infinitetrooper) [reply website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md]: Updated the text.

File: website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md (hunk 2 above)
Comment (infinitetrooper) [reply website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md]: Made the changes.

**Review by nayan-rafiq — APPROVED** [review]
LGTM
